src/lib/utils/index.ts

```diff
@@ -37,10 +37,33 @@ export const formatAmount = (amount: number, digits: number = 2) =>
     minimumFractionDigits: digits,
   })
 
+export const formatDollarAmount = (
+  amount: number | null | undefined,
+  isCompact: boolean = false,
+) => {
+  if (amount === null || amount === undefined) return ''
+
+  return Intl.NumberFormat('en', {
+    notation: isCompact ? 'compact' : undefined,
+    style: 'currency',
+    currency: 'USD',
+    minimumFractionDigits: 2,
+    maximumFractionDigits: 2,
+  }).format(amount)
+}
+
 export function formatWei(wei: bigint, units: number = 18): string {
   return formatUnits(wei, units)
 }
 
+export function formatWeiAsNumber(
+  wei: bigint | undefined,
+  units: number = 18,
+): number | undefined {
+  if (wei === undefined) return undefined
+  return Number(formatUnits(wei, units))
+}
```

Comment:
Wonder if this function is necessary, if you could just use `formatWei` and wrap it in number?

Reply:
not necessary, but simplifies these usages quite a bit: 
<img width="563" alt="image" src="https://github.com/IndexCoop/index-app/assets/136734776/4123a74d-9a90-48e6-88cb-1df167cc6c1b">